### PR TITLE
DevWs for SHINA SYSTEM containing containing SiHas Multipurpose Sensor

### PR DESCRIPTION
**Pull request by shinasys**

### CODE REVIEW

**Comment by shinasys** [comment]
@KKlimczukS @PKacprowiczS Are you reviewing?

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy [hunk 1]
```diff
@@ -29,12 +29,13 @@ metadata {
         capability "Sensor"
         capability "Contact Sensor"
         capability "afterguide46998.peopleCounter"
+        capability "afterguide46998.inOutDirection"
```
Comment (KKlimczukS) [inline devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:32]: Do other devices from this DTH (fingerprints that were added previously) support this new custom capability ? `afterguide46998.inOutDirection`

Comment (shinasys) [reply devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:32]: There is no problem simply adding a status.

File: devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy [hunk 2]
```diff
@@ -198,13 +199,26 @@ private Map getContactResult(value) {
...
+    int inout = ((int)(fpc*10).round(0))%10; // inout direction : .1 = in, .2 = out, .0 = ready
+    if(inout>2) inout = 2
+    String inoutString = ( (inout==1) ? "in" : (inout==2) ? "out":"ready")
+    String descriptionText1 = "${device.displayName} : $pc"
+    String descriptionText2 = "${device.displayName} : $inoutString"
+    log.debug "[$fpc] = people: $pc, dir: $inout, $inoutString"
+    
+    if((inout != "ready") && (prevInOut == inoutString)) {
+        sendEvent(name: "inOutDir", value: "ready", displayed: true)
+    }
```
Comment (KKlimczukS) [inline devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:214]: double checking -> 
Since this function already returns `inOutDir` event, why is this necessary to send this `inOutDir (ready)` event here?

Comment (shinasys) [reply devicetypes/shinasys/sihas-multipurpose-sensor.src/sihas-multipurpose-sensor.groovy:214]: "Ready" occurs 5 seconds after the "In" or "Out" event occurs.
So I inserted "Ready" when the same direction event occurs within 5 seconds.

**Review by KKlimczukS — APPROVED** [review]
